internal/cli/serverless/export/download.go

```diff
@@ -206,88 +217,55 @@ func DownloadCmd(h *internal.Helper) *cobra.Command {
 	downloadCmd.Flags().StringP(flag.ClusterID, flag.ClusterIDShort, "", "The cluster ID of the export.")
 	downloadCmd.Flags().String(flag.OutputPath, "", "Where you want to download to. If not specified, download to the current directory.")
 	downloadCmd.Flags().BoolVar(&force, flag.Force, false, "Download without confirmation.")
+	downloadCmd.Flags().Int(flag.Concurrency, 3, "Download concurrency.")
 	downloadCmd.MarkFlagsRequiredTogether(flag.ExportID, flag.ClusterID)
 	return downloadCmd
 }
 
-func DownloadFiles(h *internal.Helper, urls []*exportModel.V1beta1DownloadURL, path string) error {
-	if path == "" {
-		path = "."
+func DownloadFilesPrompt(h *internal.Helper, urls []*exportModel.V1beta1DownloadURL, path string, concurrency int) error {
+	if concurrency <= 0 {
+		concurrency = 3
 	}
```

Comment:
necessary? If it is, make `3` a constant, and seems `DownloadFilesWithoutPrompt` doesn't have it.